### DISCUSSION TITLE
After "wails build" my .app file just crashes immediately

### DISCUSSION THREAD

**Discussion by udfalkso** [wailsapp/wails#1998] [Q&A]
Hi All.

I'm on an intel mac.  When I run with "wails dev" the app works great.  When I try "wails build" I get a .app file but it crashes immediately upon opening.  

Am I doing something wrong?  Is there a way to see some logs and figure out what's crashing it?  

Thanks!!

```
Wails CLI v2.1.0

App Type: 		desktop
Platforms: 		darwin/amd64
Compiler: 		/usr/local/go/bin/go
Skip Bindings: 		false
Build Mode: 		Production
Obfuscated: 		false
Skip Frontend: 		false
Compress: 		false
Package: 		true
Clean Build Dir: 	false
LDFlags: 		""
Tags: 			[]
Race Detector: 		false

Building target: darwin/amd64
-----------------------------
  - Generating bindings: Done.
  - Installing frontend dependencies: Done.
  - Compiling frontend: Done.
  - Compiling application: Done.
  - Packaging application: Done.
Built '/Users/udi/HTML/searchem_go/app/build/bin/searchem.app/Contents/MacOS/searchem' in 7.289s.
```

**Comment by leaanthony** [comment]
`wails build -debug` and run it from a terminal

**Comment by udfalkso** [comment]
Thanks @leaanthony.  I gave that a shot.  After doing a debug build:
- When I open the myapp.app file in `app/build/bin/` from Finder it just immediately closes/crashes
- If I open the executable in `app/build/bin/myapp.app/Contents/MacOS` with `./myapp` then it works!  I see the expected output in the terminal and the GUI opens up

How do I make the bundled .app file work too?
